docs/source/user_guide/troubleshooting.rst

```diff
@@ -35,12 +35,30 @@ If an error lists missing modules, see the compatibility paragraph of :ref:`_ref
 The module `ansys.grpc.dpf <https://pypi.org/project/ansys-grpc-dpf/>`_ should always be synchronized with its server
 version.
 
+Using the Model
+---------------
+
+Invalid UTF-8 warning/issue 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+An error similar to the following one can be raised trying to access the py:class:`Model<ansys.dpf.core.model.Model>`:
+
+.. code-block:: default
+
+    [libprotobuf ERROR C:\.conan\897de8\1\protobuf\src\google\protobuf\wire_format_lite.cc:578] 
+    String field 'ansys.api.dpf.result_info.v0.ResultInfoResponse.user_name' contains invalid UTF-8 
+    data when serializing a protocol buffer. Use the 'bytes' type if you intend to send raw bytes.
+
+This will prevent the model to be accessed. To avoid a such inconvenience, please ensure to work with 
+a version higher than 0.3.2 of the ansys-dpf-core module: a warning will still be raised, but it should not 
+prevent to use the Model anymore. 
```

Comment:
Could you please add:

With result files reproducing this issue, to avoid the warning to pop up, you can use:
```
from ansys.dpf import core as dpf
dpf.settings.set_dynamic_available_results_capability(False)
```
However, this will disable the reading and generation of the available results of the model: static prewritten available results will be used instead.

Reply:
@cbellot000 That's done, thank you !